pom.xml

```diff
@@ -2449,6 +2450,11 @@
           </exclusion>
         </exclusions>
       </dependency>
+      <dependency>
+        <groupId>commons-cli</groupId>
+        <artifactId>commons-cli</artifactId>
+        <version>1.4</version>
```

Comment:
BTW @sarutak, why is t 1.4? I just took a quick look and we pick it as 1.2: https://github.com/apache/spark/blob/master/dev/deps/spark-deps-hadoop-3.2-hive-2.3#L29

Reply:
Hmm, I just chose the latest version.
But it might be better to choose 1.2 for the behavior compatibility.
What do you think?

Reply:
Yeah, I think 1.2 is fine. We can upgrade it in another PR :-).

Reply:
O.K, let's choose `1.2` to focus on the issue for now.